components/eamxx/src/physics/mam/CMakeLists.txt

```diff
@@ -1,11 +1,53 @@
+if (SCREAM_CIME_BUILD)
+  # PROJECT_SOURCE_DIR is SCREAM_ROOT/components
+  set(EXTERNALS_SOURCE_DIR "${PROJECT_SOURCE_DIR}/../externals")
+else()
+  # PROJECT_SOURCE_DIR is SCREAM_ROOT/components/eamxx
+  set(EXTERNALS_SOURCE_DIR "${PROJECT_SOURCE_DIR}/../../externals")
+endif()
+
+# Haero (Kokkos-based aerosol interface library)
+set(HAERO_ENABLE_GPU ${Kokkos_ENABLE_CUDA}) # might need to include HIP, etc
```

Comment:
Do you recommend to use `EAMXX_ENABLE_GPU` instead of `Kokkos_ENABLE_CUDA`?

Reply:
If you want to cover all GPU backend cases, then yes. If HAERO_ENABLE_GPU is an option for CUDA-specific stuff then no (and also change the name of the Haero option :) ).

Reply:
I am not 100% sure that we have tested all the GP backends in Haero. @jeff-cohere, any thoughts?

Reply:
I think we should go with @bartgol 's suggestion and use `EAMXX_ENABLE_GPU`. `HAERO_ENABLE_GPU` should be a proxy for all backends, not just CUDA. We just started with CUDA because we didn't have any other GPU hardware for testing.